layout/js/map.js

```diff
@@ -50,10 +47,28 @@ var animate_bars = function(data) {
     var bar = $("#bar-" + myYear);
     if (bar !== undefined) {
       bar.css(style);
+      if (myYear !== year) {
+        bar.css('opacity','0.25');
+      } else {
+        bar.css('opacity','1.0');
+      }
     }
   });
 };
 
+
+var setStateValue = function(state) {
+  $.get( "js/scaleFactors.json", function( data ) {
+      var allData = transformData["totState"][year][category];
+      for (var i = 0; i < allData.length; i++) {
+        if (allData[i]['state_name'] === state){
+          stateVal = allData[i]['value'];
+          break;
```

Comment:
To avoid a `break` here, I'd probably wrap this bit in a function and return stateVal.